public/scripts.js

```diff
@@ -151,9 +86,11 @@ $("#addDataToArray").on("click", function () {
       type_number: tmp_type_number,
       price: parseInt(tmp_price)
     };
-    simulators.push(simulator);
-    buildTable(simulators);
-    $("div").children().eq(2).children('input').val('');
+    ajaxReq('POST', '/api/simulators', JSON.stringify(simulator))

```

Comment:
A stringify-t is az ajaxolónak kell megoldania belül, hiszen meg tudja nézni, hogy get vagy post...